src/LINEBot.php

```diff
@@ -878,21 +879,51 @@ public function getNarrowcastProgress($requestId)
      */
     public function createAudienceGroupForUpdatingUserIds(
         $description,
-        $audiences,
+        $audiences = [],
```

Comment:
The audiences property has been changed from required to optional for the following endpoint.